tools/devops/automation/scripts/run-generator-tests-on-windows.ps1

```diff
@@ -1,5 +1,5 @@
 # Dump the environment to see what we're working with.
-& "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY\xamarin-macios\tools\devops\automation\scripts\show_env.ps1"
+& "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY\xamarin-macios\tools\devops\automation\scripts\show_bot_info.ps1"
```

Comment:
```suggestion
& "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY\$Env:BUILD_REPOSITORY_NAME\tools\devops\automation\scripts\show_bot_info.ps1"
```

Reply:
`$Env:BUILD_REPOSITORY_NAME` is `xamarin/xamarin-macios`, so this replacement work as-is.

Reply:
I've updated all of these to not be hardcoded anymore.